### PR TITLE
Comments on everything so far

### PR DESCRIPTION
**Pull request by Greg-Smulko**

### CODE REVIEW

File: easy/numbers/69.sqrt(x).test.js [hunk 1]
```diff
@@ -1,41 +0,0 @@
-const sqrt = require("./69.sqrt(x)");
-
-test("returns square root of 0", () => {
-  expect(sqrt(0)).toBe(0);
-});
-
-test("returns square root of 1", () => {
-  expect(sqrt(1)).toBe(1);
-});
```
Comment (Greg-Smulko) [inline easy/numbers/69.sqrt(x).test.js:9]: Take a look at "test cases", e.g. https://dev.to/bgord/simplify-repetitive-jest-test-cases-with-test-each-310m

File: easy/strings/20.ValidParentheses.js [hunk 2]
```diff
@@ -1,26 +0,0 @@
-function isValid(string) {
-  if (string.length % 2 === 1) return false;
```
Comment (Greg-Smulko) [inline easy/strings/20.ValidParentheses.js:2]: Nice! Well spotted! Nice optimization!

It's not the case here obviously, but just bear in mind, that if getting a length from an input may be expensive (e.g. `IEnumerable` in .NET, where you need to iterate through the whole collection to get its size (possibly even by loading elements from the DB) or even not possible (for unbounded streams of data, think live video ;) ).

File: easy/strings/20.ValidParentheses.js [hunk 3]
```diff
@@ -1,26 +0,0 @@
-function isValid(string) {
```
Comment (Greg-Smulko) [inline easy/strings/20.ValidParentheses.js:1]: Please don't use `string` as a variable name, even if it's allowed in JS.
In most languages, it's the reserved keyword for a type name. 
Perhaps `input` instead?

File: easy/strings/20.ValidParentheses.js [hunk 4]
```diff
@@ -1,26 +0,0 @@
...
-    "{": "}",
-    "[": "]",
-  };
-  const stackOfBrackets = [];
-  for (let i = 0; i < string.length; i++) {
-    if (matchingSetsOfParentheses[string[i]]) {
-      stackOfBrackets.push(string[i]);
-    } else {
-      const lastBracketInStack = stackOfBrackets.pop();
-      if (matchingSetsOfParentheses[lastBracketInStack] !== string[i]) {
```
Comment (Greg-Smulko) [inline easy/strings/20.ValidParentheses.js:14]: Extract `string[i]` to a var?

File: package.json [hunk 5]
```diff
@@ -1,24 +0,0 @@
-{
-  "name": "leetcode",
-  "version": "1.0.0",
-  "main": "index.js",
-  "dependencies": {
-    "eslint": "^7.27.0",
-    "jest": "^27.0.2"
-  },
-  "devDependencies": {},
```
Comment (Greg-Smulko) [inline package.json:9]: These two are semantically `devDependencies`.

**Review by Greg-Smulko — COMMENTED** [review]
Nice overall! :D

File: package.json (hunk 5 above)
Comment (mosmulko) [reply package.json:9]: Well spotted, thank you. Pretty sure I uninstalled them and then installed them again with -D, but I had some troubles with mixed master and main branches, so I must have pulled the old package.json file when I deleted the master branch.

Comment (Greg-Smulko) [reply package.json:9]: 👍 
The `main` branch, not `master` - we're being inclusive, aren't we? :)

Comment (Greg-Smulko) [reply package.json:9]: Ah, ok, sorry, I reread it and now know what you mean. :) 
👍

**Pull request closed without merging**